pkg/services/query/query_fetcher.go

```diff
@@ -0,0 +1,34 @@
+package query
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/qustavo/dotsql"
+)
+
+// GetQueryStringFromFile accepts a query name,
+// returns SQL query as a string
+func GetQueryString(queryName string) (string, error) {
+	sqlDir := "services/query/sql_scripts"
```

Comment:
This needs to be defined as a constant in the query package rather than in the function that uses it since it's not going to change.

```suggestion
const sqlDir string = "services/query/sql_scripts"

// GetQueryStringFromFile accepts a query name,
// returns SQL query as a string
func GetQueryString(queryName string) (string, error) {

```

Reply:
Good call, I've made this change!